kernel/src/Plugins/FakePkg/FakePkgManager.cpp

```diff
@@ -209,6 +209,16 @@ bool FakePkgManager::ShellCorePatch()
         return false;
     }
 
+	
+#if MIRA_PLATFORM == MIRA_PLATFORM_ORBIS_BSD_672
```

Comment:
What does this do exactly, and it will need to be found/support added for *all* firmwares